cypress/common/selectors.ts

```diff
@@ -59,7 +60,11 @@ export const fields = {
 
 export const buttons = {
     buttonPrimary: '.btn-primary',
-    buttonDanger: '.btn-danger'
+    buttonDanger: '.btn-danger',
+    sharedStoreCrossFrameworkAppActionsButtons: [
+        '[data-e2e="DECREMENT_BUTTON"]',
+        '[data-e2e="INCREMENT_BUTTON"]'
+    ]
```

Comment:
```suggestion
    sharedStoreCrossFrameworkAppActionsButtons: {
        decrement: '[data-e2e="DECREMENT_BUTTON"]',
        increment: '[data-e2e="INCREMENT_BUTTON"]'
    }
```

I think it will look better in test

instead

`selector: buttons.sharedStoreCrossFrameworkAppActionsButtons[1]`

to 

`selector: buttons.sharedStoreCrossFrameworkAppActionsButtons.increment`

What do you think?

Reply:
fixed